operations/tasks/upgrade-docker.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Mark node as unschedulable
+  command: >
+    oadm manage-node {{ inventory_hostname }} --schedulable=false
+
+- name: Wait until no more pods are running in node
+  register: active_pods_result
+  shell: >
+    oadm manage-node {{ inventory_hostname }} --list-pods |
+    grep -v 'STATUS' |
+    grep -v 'Completed' |
+    grep -v 'Error' |
+    wc -l
+  until: '0' in active_pods_result.stdout_lines
+  changed_when: false # read-only command
+  environment: "{{ osbs_environment }}"
+  retries: "{{ osbs_wait_active_pods_retries }}"
+  delay: "{{ osbs_wait_active_pods_delay }}"
+
+- name: Update docker package
+  yum:
+    name: "{{ osbs_docker_version }}"
```

Comment:
Note if one is using Atomic Host, this isn't necessary as upgrades are in the background (and also atomic, etc.)

Reply:
Thanks for the feedback!

OSBS doesn't use atomic host. Ideally, this would be handled by `openshift-ansible`, but AFAIK it only provides OpenShift upgrades (which may upgrade docker as a side effect). Our use case is to update docker without necessarily updating OpenShift.

We also want to do this with minimal disruption.